pkgs/by-name/lo/local-ai/package.nix

```diff
@@ -371,18 +373,18 @@ let
       stdenv;
 
   pname = "local-ai";
-  version = "2.12.4";
+  version = "2.13.0";
   src = fetchFromGitHub {
     owner = "go-skynet";
     repo = "LocalAI";
-    rev = "v${version}";
-    hash = "sha256-piu2B6u4ZfxiOd9SXrE7jiiiwL2SM8EqXo2s5qeKRl0=";
+    rev = "V${version}";
```

Comment:
should be `v`

Reply:
which makes me update the hash

Reply:
Thanks for testing. The project changed the tag name.
I updated the PR.